### PR TITLE
MG1933089

### PR DESCRIPTION
**Pull request by Abonnen**

### CODE REVIEW

File: MG1933089/java/InputStream_close.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+ */
+public class InputStream_close {
+    /**
+     * input: 0
+     * class0 FileInputStream$class0
+     * output: 0
+     * ret0 ret0=0
+     * InputStream$class0-File$File(String pathname);InputStream$InputStream(File file) & String$pathname="file.txt"
+     */
+    public static void close0() {
```
Comment (idiomaticrefactoring) [inline MG1933089/java/InputStream_close.java]: link

File: MG1933089/swift/Stream_close.swift [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+ */
+import Foundation
+class Stream_close{
+     /**
+     * input: 0
+     * class0 Stream$class0
+     * output: 0
+     * ret0 ret0=-1
+     * OutputStream$class0-OutputStream$init?(toFileAtPath path: String, append shouldAppend: Bool) & String$path="file.txt";Bool$shouldAppend=false 
+     */
```
Comment (idiomaticrefactoring) [inline MG1933089/swift/Stream_close.swift]: link

File: MG1933089/swift/String_suffix_String$Index__String_firstIndex_Character.swift [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    /**
+     *input : 3
+     *class0 class0="hello"
+     *from String.Index$from
+     *of of="l"
+     *ouput : 2
+     *ret0 ret0="hello"
+     *ret1 String.Index$ret1
+     *String.Index$from-String$index(_ i: String.Index, offsetBy n: String.IndexDistance) & String.Index=String$startIndex;offsetBy=0
+     */
```
Comment (idiomaticrefactoring) [inline MG1933089/swift/String_suffix_String$Index__String_firstIndex_Character.swift]: link

**Review by idiomaticrefactoring — COMMENTED** [review]
good+2

**Comment by Abonnen** [comment]
InputStream,6 change
PrintStream,30 same
String,42 same

**Pull request closed without merging**